packages/flutter_tools/lib/src/base/dds.dart

```diff
@@ -55,15 +50,15 @@ class DartDevelopmentService {
       _ddsInstance = await ddsLauncherCallback(
           observatoryUri,
           serviceUri: ddsUri,
-          enableAuthCodes: !disableServiceAuthCodes,
-          ipv6: ipv6,
+          enableAuthCodes: disableServiceAuthCodes == true,
```

Comment:
looking at the diff, should this not be `== false`?

Reply:
Yes, and a bunch of tests failed.  Fixed!

Reply:
yay tests